src/components/comment/index.js

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import { useState } from "react";
+import { Button } from "@material-ui/core";
+import "./style.css";
+import ThumbUpAltIcon from "@material-ui/icons/ThumbUpAlt";
+import ThumbDownIcon from "@material-ui/icons/ThumbDown";
+import AccountCircleTwoToneIcon from "@material-ui/icons/AccountCircleTwoTone";
+import Box from "@material-ui/core/Box";
+
+function Comment(props) {
+  const [tds, setTds] = useState("inherit");
+  const [tus, setTus] = useState("inherit");
+  const [tuc, setTuc] = useState(0);
+  const [tdc, setTdc] = useState(0);
+
+  function handleClick() {
+    console.log("Click happened");
+  }
+
+  return (
+    <div>
+      <div className="author-details">
+        <Box m={0} p={0.6}>
+          <AccountCircleTwoToneIcon fontSize="large" color="primary" />
+        </Box>
+        Bill Joy{props.author}
+      </div>
+      <div className="com-text">
+        {props.text}Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
+        do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+        ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+        aliquip
+      </div>
+      <div className="menu">
+        <Box m={0.5} p={0.8}>
+          <Button
+            variant="contained"
+            color="primary"
+            size="small"
+            onClick={handleClick()}
```

Comment:
the onClick handler must be passed a function. You actually called the function before passing so this won't work as you expect it to.

```suggestion
            onClick={handleClick}
```